### ISSUE TITLE
Not ignoring highway=construction

### ISSUE THREAD

**Issue by Phyks** [poutnikl/Car-Profile#5]
Hi,

I'm not sure whether this is on purpose or not but it seems these profiles are not ignoring (or putting penalties on) `highway=construction` contrary to the trekking (huge penalty) or the walking (muddy penalty) profiles.

Thanks!

**Comment by poutnikl** [comment]
Yes, it is on purpose. Note that this tag is supposed for road being built, not for short term closures.

But I currently consider my car profiles, not based on the newer Arndt's kinematic model, as obsolete. The exception can be toll-free variants(either based oh this model or not), as the new built-in profiles,at least until recently, do not consider toll-free policy.

**Comment by Phyks** [comment]
Ok, thanks for the feedback on this. As in the other issue, this might be something worth adding in the README or somewhere visible, I can write a PR for this if you want.

> Note that this tag is supposed for road being built, not for short term closures.

Usage seems to depend. It is indeed used for roads being built but I also often see it used for roads under heavy works (which could be argued to be a road being destroyed / rebuilt) which will change the configuration of the street.

I'm not sure to fully understand why this is on purpose though, as a route currently under construction is likely to not be driveable.

**Comment by poutnikl** [comment]
And that is why it is excluded, afaik in my car profiles.
OTOH, for trekking bike purposes, it may be advantage to to use them under "emergency route mode" , with relaxed rules following, either in saddle either pushing.

**Comment by Phyks** [comment]
Oh ok, that might be another case of different versions of the profiles across repositories. I totally agree it should be excluded for cars but could be used as emergency route for trekking.

So, it is indeed in the profile from this repository: https://github.com/poutnikl/Car-Profile/blob/master/Car-test-Template.brf#L267.

But when looking at the `Brouter-profiles` repository, I cannot find it in the car profiles (https://github.com/poutnikl/Brouter-profiles/blob/master/CarProfiles/Car-Eco.brf for instance). Note that the header here is one version older than the `Car-Profile` repository.

**Comment by poutnikl** [comment]
It is possible, as I do not automatically create new profiles after each master update, so even master template may be ahead.
But I rather cease car profile development as the kinematic model of new built-in profile seems better ( but one may want toll-free versions). I have somewhere there these toll-free variants.
